docs/modules/ROOT/pages/cli/cli.adoc

```diff
@@ -8,6 +8,16 @@ Releases of the Camel K CLI are available on:
 - Github Releases: https://github.com/apache/camel-k/releases
 - Homebrew (Mac and Linux): https://formulae.brew.sh/formula/kamel
 
+== Running on macOS
+
+Before running the CLI on macOS, it may be necessary to give adequate permissions for it to run.
+
+It can either be done on the "Privacy & Security" panel in the System Settings or via command-line:
+
+```
+xattr -dr com.apple.quarantine ~/path/to/kamel
```

Comment:
I'm just wondering if we need the `r` flag?

Reply:
I've just tested here and we don't need that. 👍